operator/github/v0/README.mdx

```diff
@@ -0,0 +1,266 @@
+---
+title: "GitHub"
+lang: "en-US"
+draft: false
+description: "Learn about how to set up a VDP GitHub component https://github.com/instill-ai/instill-core"
+---
+
+The GitHub component is an operator component that allows users to do anything available on GitHub.
+It can carry out the following tasks:
+
+- [Get All Pull Requests](#get-all-pull-requests)
+- [Get Pull Request](#get-pull-request)
+- [Get Commit](#get-commit)
+- [Get Review Comments](#get-review-comments)
+- [Create Review Comment](#create-review-comment)
+- [Get All Issues](#get-all-issues)
+- [Get Issue](#get-issue)
+- [Create Issue](#create-issue)
+- [Create Webhook](#create-webhook)
+
+
+
+## Release Stage
+
+`Alpha`
+
+
+
+## Configuration
+
+The component configuration is defined and maintained [here](https://github.com/instill-ai/component/blob/main/application/github/v0/config/definition.json).
+
+
+
+
+## Setup
+
+
+| Field | Field ID | Type | Note |
+| :--- | :--- | :--- | :--- |
+| Token | `token` | string | Fill in your GitHub access token for advanced usages. For more information about how to create tokens, please refer to the https://github.com/settings/tokens. |
+
+
+
+
+## Supported Tasks
+
+### Get All Pull Requests
+
+Get the list of all pull requests in a repository
+
+
+| Input | ID | Type | Description |
+| :--- | :--- | :--- | :--- |
+| Task ID (required) | `task` | string | `TASK_GET_ALL_PULL_REQUESTS` |
+| Owner (required) | `owner` | string | owner of the repository |
+| Repository (required) | `repository` | string | repository name |
+| State (required) | `state` | string | State of the PRs, including open, closed, all. Default: open |
+| Sort (required) | `sort` | string | Sort the PRs by created, updated, popularity, or long-running. Default: created |
+| Direction (required) | `direction` | string | Direction of the sort, including asc or desc. Default: asc |
```

Comment:
We don't need to specify default values for required arguments as the user will have to set these themselves. Are these meant to be optional arguments?